config/opal_config_pmix.m4

```diff
@@ -63,6 +63,8 @@ AC_DEFUN([OPAL_CONFIG_PMIX], [
 
     OPAL_3RDPARTY_WITH([pmix], [pmix], [package_pmix])
 
+    OMPI_PMIX_ADD_ARGS
```

Comment:
This should be inside the m4_ifdef on line 71; we should not add PMIx subconfigure args if we did not set package_pmix.

Reply:
done